tests/acceptance/features/apiGraph/addUserToGroup.feature

```diff
@@ -203,3 +203,39 @@ Feature: add users to group
       | Brian    |
       | Carol    |
     Then the HTTP status code should be "404"
+
+
+  Scenario: try to add users to a group twice
+    Given the administrator has given "Alice" the role "Admin" using the settings api
+    And these users have been created with default attributes and without skeleton files:
+      | username |
+      | Brian    |
+      | Carol    |
+    And user "Alice" has created a group "grp1" using the Graph API
+    And the administrator "Alice" has added the following users to a group "grp1" at once using the Graph API
+      | username |
+      | Brian    |
+      | Carol    |
+    When the administrator "Alice" adds the following users to a group "grp1" at once using the Graph API
+      | username |
+      | Brian    |
+      | Carol    |
+    Then the HTTP status code should be "400"
+    And the following users should be listed in the following groups
+      | username | groupname |
+      | Brian    | grp1      |
+      | Carol    | grp1      |
```

Comment:
This seems strange 400 status code giving the reponse that contains the above data.

Reply:
@SagarGi  plz look at the step implementation https://github.com/owncloud/ocis/blob/2bf458385d7ffeb015066211e80b5eb33af299ee/tests/acceptance/features/bootstrap/GraphContext.php#L1181 
we are doing getMembersList api request to get list of group separately in last step